src/Store/StoreThread.re

```diff
@@ -368,6 +369,8 @@ let start =
     window,
   );
 
+  let signalHandle = Luv.Signal.init() |> Result.get_ok;
+
```

Comment:
This doesn't seem to be used anywhere. Assuming it's left over form the `SIGINT` handler you were investigating?

Reply:
Ah yeah thanks! I thought I had run `esy b dune build @check`, but I guess I hadn't.